cmd/evm/runner.go

```diff
@@ -227,6 +227,7 @@ func runCmd(ctx *cli.Context) error {
 		EVMConfig: vm.Config{
 			Tracer: tracer,
 		},
+		Random: &genesisConfig.Mixhash,
```

Comment:
Hmmm, any particular reason for picking Mixhash to init the random field with?

Reply:
Sorry, no particular reason. I just see there are two fields in the genesis config, `Mixhash` and `ParentHash`. Their type is `common.Hash`, the same as `Random`. Any suggestions about which field should be used?